themes/10up-theme/package.json

```diff
@@ -16,7 +16,7 @@
     "node": ">=12.0.0"
   },
   "devDependencies": {
-    "10up-toolkit": "^2.0.0",
+    "10up-toolkit": "^2.1.0",
```

Comment:
@willhowat 

I think we are at 3.0.0 now. 
We should verify and test before pushing through.

Reply:
There's an open PR for the 3.0 upgrade #87 so we can remove this change here (unless it's necessary)

Reply:
@joesnellpdx wp-scaffold trunk is currently dependent on 10up-toolkit ^2.1.0 so this aligns. I think I probably made the change rather than merging in. Would it be OK to resolve this on that basis?

Reply:
up to 4.1.1 now